checkstyle.xml

```diff
@@ -3,6 +3,10 @@
 <module name="Checker">
     <module name="FileTabCharacter" />
     <module name="NewlineAtEndOfFile" />
+    <module name="RegexpMultiline">
```

Comment:
Please do not use this module 😅
The proper way is `.gitattributes` with `git add --renormalize .` 😇

Reply:
See https://github.com/jenkinsci/vstestrunner-plugin/pull/16 for reference

Reply:
I am not a big fan of automatic conversions. But I implemented your suggestion.